lib/kubernetes-deploy/runner.rb

```diff
@@ -131,6 +138,8 @@ def run(verify_result: true, allow_protected_ns: false, prune: true)
       success = false
     ensure
       @logger.print_summary(success)
+      tags = %W(namespace:#{@namespace} sha:#{ENV['REVISION']})
```

Comment:
Explained in https://github.com/Shopify/kubernetes-deploy/pull/118#discussion_r126109263